src/main/java/com/ksm/wstbackoffice/exception/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package com.ksm.wstbackoffice.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.Date;
+
+@ControllerAdvice
+public class ControllerExceptionHandler {
+    @ExceptionHandler(ResourceNotFoundException.class)
+    public ResponseEntity<ErrorResponseMessage> resourceNotFoundException(ResourceNotFoundException ex, WebRequest request) {
+        ErrorResponseMessage message = new ErrorResponseMessage(
+                HttpStatus.NOT_FOUND.value(),
+                new Date(),
+                ex.getMessage(),
+                request.getDescription(false));
```

Comment:
What this information is? Can you give an example?

Reply:
Request:
GET http://localhost:8081/api/countries/QQQ

Response:
{
  "statusCode": 404,
  "timestamp": "2021-01-03T19:21:17.497+00:00",
  "message": "Not found country with id QQQ ",
  "description": "uri=/api/countries/QQQ"
}

Reply:
where did you find this example?
If somebody does GET /api/countries/QQQ
You will answer with the message with the description /api/countries/QQQ
Why the client needs this information. The client knows the URI it used to do request

Reply:
Changed to

{
  "localDate": "2021-01-03",
  "message": "Not found country with id QQQ "
}